projects/plugins/jetpack/extensions/blocks/payment-buttons/editor.scss

```diff
@@ -2,7 +2,7 @@
 	--jetpack-payment-buttons-gap: 0.5em;
 	&.is-layout-flex {
 		gap: var(--jetpack-payment-buttons-gap);
-		display: block;
```

Comment:
I found that this change was necessary. Otherwise, the editor controls, like justification and orientation, were ignored. 

I configured a row with a 70% button and 30% button. In prod I see this:


<img width="1205" alt="Screenshot 2567-08-22 at 16 13 20" src="https://github.com/user-attachments/assets/7a2619df-0684-44fb-a0ef-2d0ba604556d">

If I change the `display` value to `flex` it fixes the issue:
<img width="1217" alt="Screenshot 2567-08-22 at 16 13 30" src="https://github.com/user-attachments/assets/27526b41-ba0f-489c-937d-8183cf43ce32">

CC @dsas who last edited this file.